docs/_blog/2024_q3_update.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "2024 Q3 Update: What Have We Been Up To?"
+description: "2024 Q3 Update on the Bitcoin Dev Kit Project"
+authors:
+    - thunderbiscuit
+date: "2024-11-01"
+tags: ["BDK", "project"]
+draft: false
+---
+
+### Q3 2024: What Have We Been Up To?
+The bitcoindevkit team has been hard at work for Q3 in 2024, polishing the API of our `bdk_wallet` crate and releasing 4 new beta versions (1, 2, 3, and 4!), and aiming to release a final 1.0 release by the end of 2024. Here are some of the notable changes and upgrades to the software libraries we maintain:
+- **RBF by default on TxBuilder.** The transaction builder in BDK will now signal RBF by default.
+- **New wallet builder API.** The new wallet builder offers flexibility and ease-of-development for future features. We've also been listening to user feedback, and brought back support for single-descriptor wallets.
+- **MVP of the Book of BDK.** We are working on a high-level documentation website for BDK libraries called the Book of BDK. The MVP website is live at [bookofbdk.com](https://bookofbdk.com/).
+- **Bug chasing and optimizations.** As feedback from early testers comes in, we are keeping a close eye on reported bugs and questions, and have been fixing a ton of smaller but very important snags!
+- **Development of a CBF client create and related bindings.** Work is ongoing on a crate to allow BDK users to interoperate with a new CBF library called [Kyoto](https://github.com/rustaceanrob/kyoto). Work has been done to integrate this with the language bindings for mobile users, and the preliminary integrations have been very positive.
```

Comment:
spelling: I think `create` is supposed to be `crate`?

Reply:
yep.